QEfficient/cloud/infer.py

```diff
@@ -78,8 +81,12 @@ def main(
     )
 
     # Handle qpc generation
+
     if qpc_exists(qpc_dir_path):
         logger.info(f"Pre-compiled qpc found at {qpc_dir_path}! Executing with given prompt")
+
+        compile_time = "NA"
+
```

Comment:
can you remove this, it's not required. as when benchmark=True, this variable will always exist

Reply:
This assignment is needed for when the model was compiled before. I changed the assignment as compile_time = "pre-compiled"
Please see https://github.com/quic-morteza/efficient-transformers/blob/8f6abc5a57bd369c4deb1cea8d18725ddc57e2a2/QEfficient/cloud/infer.py#L92